webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTableHelper.js

```diff
@@ -26,3 +29,50 @@ export const formatValue = variable => {
       return value;
   }
 };
+
+const joinErrors = errors => errors.map(err => err.message).join(', ');
+
+const onCompleted = (toggleModal, showToast) => data => {
+  toggleModal();
+  const { errors } = data.deleteAnsibleVariableOverride;
+  if (Array.isArray(errors) && errors.lenght > 0) {
+    showToast({
+      type: 'error',
+      message: formatError(joinErrors(errors)),
+    });
+  } else {
+    showToast({
+      type: 'success',
+      message: __('Ansible variable override was successfully deleted.'),
+    });
+  }
+};
+
+export const findOverride = (variable, hostname) =>
+  variable.lookupValues.nodes.find(
+    item =>
+      item.value === variable.currentValue.value &&
+      item.match === `fqdn=${hostname}`
+  );
+
+const formatError = error =>
+  sprintf(
+    __(
+      'There was a following error when deleting Ansible variable override: %s'
+    ),
+    error
+  );
```

Comment:
Why not call `sprintf` directly?

Reply:
So that the error message text is defined in one place, `formatError` is called in 2 places